dockerfiles/state-signals-exporter-0.5.2-pod/state_signals_exporter.py

```diff
@@ -0,0 +1,60 @@
+
+import state_signals
+import time
+import argparse
+
+
+class StateSignalsExporter:
+    """
+    This class is run state signals exporter
```

Comment:
"This class runs the state signals exporter"

Reply:
Fixed